commit aa601d024424c40ae9a69b0c4e394a70ea0570c8
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Mon Jan 24 19:33:43 2022 -0800

    x86: Use CHECK_FEATURE_PRESENT to check HLE [BZ #27398]
    
    HLE is disabled on blacklisted CPUs.  Use CHECK_FEATURE_PRESENT, instead
    of CHECK_FEATURE_ACTIVE, to check HLE.
    
    (cherry picked from commit 501246c5e2dfcc278f0ebbdb72345cdd239521c7)

diff --git a/sysdeps/x86/tst-cpu-features-supports.c b/sysdeps/x86/tst-cpu-features-supports.c
index 9d76e6bd3f8db024..faa5091b78431487 100644
--- a/sysdeps/x86/tst-cpu-features-supports.c
+++ b/sysdeps/x86/tst-cpu-features-supports.c
@@ -130,7 +130,7 @@ do_test (int argc, char **argv)
   fails += CHECK_FEATURE_ACTIVE (gfni, GFNI);
 #endif
 #if __GNUC_PREREQ (11, 0)
-  fails += CHECK_FEATURE_ACTIVE (hle, HLE);
+  fails += CHECK_FEATURE_PRESENT (hle, HLE);
   fails += CHECK_FEATURE_PRESENT (ibt, IBT);
   fails += CHECK_FEATURE_ACTIVE (lahf_lm, LAHF64_SAHF64);
   fails += CHECK_FEATURE_PRESENT (lm, LM);
